plugins/cpp/parser/src/diagnosticmessagehandler.cpp

```diff
@@ -8,17 +8,19 @@ namespace parser
 {
 
 DiagnosticMessageHandler::DiagnosticMessageHandler(
+  llvm::raw_ostream& os,
+  clang::DiagnosticOptions* diags,
   SourceManager& srcMgr_,
   std::shared_ptr<odb::database> db_)
-    : _srcMgr(srcMgr_), _db(db_)
+    : TextDiagnosticPrinter(os, diags), _srcMgr(srcMgr_), _db(db_)
```

Comment:
```suggestion
    : TextDiagnosticPrinter(llvm::errs(), diags), _srcMgr(srcMgr_), _db(db_)
```
I think the first parameter of this constructor can be removed.
If you remove this, then don't forget to remove `#include <llvm/Support/raw_ostream.h>` from `diagnosticmessagehandler.h`.